### DISCUSSION TITLE
Welcome to Qwerty Learner Discussions!

### DISCUSSION THREAD

**Discussion by RealKai42** [RealKai42/qwerty-learner#105] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xiaod0ng** [comment]
Hi Kaiyi,
请问发音是自动朗读吗？https://kaiyiwing.github.io/qwerty-learner/ 此链接下不能获得发音。谢谢

**Reply by RealKai42** [reply]
是自动朗读的，请确定右上角的朗读开关是否打开
![CleanShot 2021-06-29 at 13 55 23@2x](https://user-images.githubusercontent.com/44634134/123744707-b7edfa80-d8e1-11eb-984c-d5ba68f1efef.png)

**Reply by Xiaod0ng** [reply]
该开关是打开的，英音美音均无反应。

**Reply by RealKai42** [reply]
是什么浏览器？ 尝试换成 Chrome 试一下？

**Reply by Xiaod0ng** [reply]
Chrome和firefox均无单词读音，但是有打字的提示音。

**Reply by Sakura1221** [reply]
应该是和网络有关，能优化一下吗？我这有时候发音也要等一会，有些时候干脆就没有了

**Reply by RealKai42** [reply]
发音用的是网易有道的开放 API，可能是他们那边的问题

**Reply by Sakura1221** [reply]
> 发音用的是网易有道的开放 API，可能是他们那边的问题

之后会考虑开发注册登录功能吗？记录学习数据？后续还可以考虑利用遗忘曲线提醒复习（可有可无）

**Reply by RealKai42** [reply]
暂时没有
一是服务器的成本比较高，二是最近在忙实验室的项目，腾不出时间

**Comment by Lichtung01** [comment]
学生刚好在学习Essential English Word这本书，想让他们巩固一下，请问是不是无法隐去例句中的单词，想的是听到这个单词发音，然后可以直接拼出来。底下的例句如果可以有选择，能换成这个词的英文或中文解释就好了。

另外，本想截图说明下我的意思，但发现用软件截图时，截到的内容会自动变成任意键开始，就没截成。

**Comment by Por14** [comment]
英语期考是机考来着，想在考试前苦练打字速度，但在keybr上老是过不去那几个键（咳），敲烦了就在浏览器上逛，看还有没有什么打字平台能兼顾实用与不枯燥性，哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈哈真让我找着了！很开心（且很荣幸：））能遇到这个平台！非常感谢你们所做的一切呜呜呜呜呜呜呜呜。致谢部分我也津津有味的看了！由日常实践得到的灵感延伸到一个项目以及用自己的知识去帮助自己以及大家，真的超帅！！！——
